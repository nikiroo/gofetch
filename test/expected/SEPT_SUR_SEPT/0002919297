   PROCÈS DU SOUS-MARIN DANOIS: PETER MADSEN CONDAMNÉ À PERPÉTUITÉ 

  [Monde] 26-09-2018 14:37

  o News link: http://s.7s7.be/3479969
  o Source link: 


    Par: rédaction
    
    26/09/18  -  14h37  Source: Belga    Peter Madsen et Kim Wall ©
    EPA / AP.
    
    MISE  À JOUR La Haute cour de Copenhague a confirmé mercredi en
    appel  la  condamnation de l'inventeur danois Peter Madsen à la
    réclusion  à  perpétuité  pour  l'assassinat  d'une journaliste
    suédoise à bord de son sous-marin artisanal en août 2017.
    
    © ap.
    
    © afp.
    
    © ap.
    
    © ap.
    
    © reuters.
    
    Peter  Madsen,  47 ans, avait interjeté appel de la durée de sa
    peine  de  prison  après  sa  condamnation  à la perpétuité, en
    première  instance  en  avril  dernier,  pour  le  meurtre avec
    préméditation,  précédé de violences sexuelles, de Kim Wall, 30
    ans.
    
    [ Pas d'aveux ]
    
    Sans  être passé aux aveux, défendant bec et ongles la thèse de
    la  mort  accidentelle,  il  n'a jamais pour autant contesté le
    verdict  de  culpabilité.  Son  avocate a indiqué après l'appel
    qu'elle  allait  étudier  un éventuel recours auprès de la Cour
    suprême.
    
    [ "Préméditation minitieuse" ]
    
    La   Haute   cour   a  motivé  sa  décision  par  le  caractère
    "particulièrement   brutal"  du  crime  et  "les  circonstances
    aggravantes"   que   constitue  selon  elle  sa  "préméditation
    minutieuse".  "Il  n'y  a  guère  de peine trop sévère pour cet
    homme",   a  déclaré  à  l'issue  de  l'audience  le  procureur
    Kristian Kirk.
    
    [ Rappel des faits ]
    
    Le   10  août  2017,  Peter  Madsen  avait  embarqué  dans  son
    submersible  artisanal Kim Wall, 30 ans, qui projetait d'écrire
    un  reportage  sur  ses désirs de conquête du ciel et des fonds
    marins.  Elle  avait  été  portée disparue dans la nuit par son
    compagnon  qui  a assisté au procès en appel et son corps avait
    ensuite  été  retrouvé  en  mer,  démembré.  L'autopsie  avance
    l'hypothèse  d'un  étouffement  ou  d'un  égorgement précédé de
    sévices  sexuels  mais  la  cause  exacte  du décès n'a pu être
    établie.
    
    [ Multiples blessures ]
    
    De  multiples  blessures  -  14  -  ont été identifiées dans et
    autour  des  parties  génitales de la jeune femme. En outre, le
    parquet  a  invoqué  les  outils  inutiles  sur  un  sous-marin
    embarqués  par  l'accusé peu de temps avant les faits, dont une
    scie  à  bois,  des sangles de valise et un tournevis affûté de
    50 cm de long.
    
    [ Rappel: Madsen "désolé" ]
    
    "Je  suis terriblement triste pour les proches de Kim de ce qui
    s'est  passé",  avait  déclaré  Peter  Madsen  à  la clôture de
    l'audience,  utilisant un adjectif qui en danois peut également
    signifier  "désolé". Le procureur Kristian Kirk avait requis la
    prison  à  vie. "Quand je regarde Peter Madsen (...) je ne vois
    pas  de raison pour (lui infliger) une peine plus faible que la
    perpétuité".  Selon  lui,  la  préméditation  et  la nature des
    violences  qui lui sont reprochées justifiaient son enfermement
    à vie.
    
    [ "Meurtrier sexuel cynique" ]
    
    "Nous   parlons  d'un  meurtrier  sexuel  cynique,  pervers  et
    calculateur  et  pour  cette raison je ne pense pas qu'il y ait
    une  marge  d'hésitation", avait souligné M. Kirk. "Par défaut,
    tous  les  meurtres  sont cyniques, ce n'est pas propre à cette
    affaire",  avait  rétorqué  l'avocate  de  Peter Madsen, Betina
    Hald  Engmark.  "La  défense  est  d'avis  que le bon niveau de
    peine  se situe entre 14 et 16 ans", avait-elle martelé lors de
    sa plaidoirie.



