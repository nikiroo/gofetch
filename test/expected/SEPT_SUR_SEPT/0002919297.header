0Procès du sous-marin danois: Peter Madsen condamné à perpétuité	null/SEPT_SUR_SEPT/0002919297		70
i  [Monde] 26-09-2018 14:37
i
i    La  Haute  cour  de  Copenhague a confirmé mercredi en appel la
i    condamnation  de l'inventeur danois Peter Madsen à la réclusion
i    à perpétuité pour ...
i
