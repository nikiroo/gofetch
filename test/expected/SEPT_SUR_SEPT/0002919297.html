<!DOCTYPE html>
<html>
<head>
  <meta http-equiv='content-type' content='text/html; charset=utf-8'>
  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
  <style type='text/css'>
    body { margin: 1em 15%; }
  </style>
</head>
<body>
<div class='story'>
	<h2>Procès du sous-marin danois: Peter Madsen condamné à perpétuité</h2>
	<div class='details'>([Monde] 26-09-2018 14:37)</div>
	<br/>
    <ul>
    <li>Reference: <a href=''>0002919297</a></li>
        <li>News link: <a href='http://s.7s7.be/3479969'>http://s.7s7.be/3479969</a></li>
        <li>Source link: <a href=''></a></li>
    </ul>
	<br/>
	<div class='content' style='text-align: justify'>
		Par: rédaction<br/><br/>26/09/18 - 14h37 Source: Belga    Peter Madsen et Kim Wall © EPA / AP.<br/><br/>MISE À JOUR La Haute cour de Copenhague a confirmé mercredi en appel la condamnation de l&#x27;inventeur danois Peter Madsen à la réclusion à perpétuité pour l&#x27;assassinat d&#x27;une journaliste suédoise à bord de son sous-marin artisanal en août 2017.<br/><br/>© ap.<br/><br/>© afp.<br/><br/>© ap.<br/><br/>© ap.<br/><br/>© reuters.<br/><br/>Peter Madsen, 47 ans, avait interjeté appel de la durée de sa peine de prison après sa condamnation à la perpétuité, en première instance en avril dernier, pour le meurtre avec préméditation, précédé de violences sexuelles, de Kim Wall, 30 ans.<br/><br/><h3>Pas d&#x27;aveux</h3><br/><br/>Sans être passé aux aveux, défendant bec et ongles la thèse de la mort accidentelle, il n&#x27;a jamais pour autant contesté le verdict de culpabilité. Son avocate a indiqué après l&#x27;appel qu&#x27;elle allait étudier un éventuel recours auprès de la Cour suprême.<br/><br/><h3>&quot;Préméditation minitieuse&quot;</h3><br/><br/>La Haute cour a motivé sa décision par le caractère &quot;particulièrement brutal&quot; du crime et &quot;les circonstances aggravantes&quot; que constitue selon elle sa &quot;préméditation minutieuse&quot;. &quot;Il n&#x27;y a guère de peine trop sévère pour cet homme&quot;, a déclaré à l&#x27;issue de l&#x27;audience le procureur Kristian Kirk.<br/><br/><h3>Rappel des faits</h3><br/><br/>Le 10 août 2017, Peter Madsen avait embarqué dans son submersible artisanal Kim Wall, 30 ans, qui projetait d&#x27;écrire un reportage sur ses désirs de conquête du ciel et des fonds marins. Elle avait été portée disparue dans la nuit par son compagnon qui a assisté au procès en appel et son corps avait ensuite été retrouvé en mer, démembré. L&#x27;autopsie avance l&#x27;hypothèse d&#x27;un étouffement ou d&#x27;un égorgement précédé de sévices sexuels mais la cause exacte du décès n&#x27;a pu être établie.<br/><br/><h3>Multiples blessures</h3><br/><br/>De multiples blessures - 14 - ont été identifiées dans et autour des parties génitales de la jeune femme. En outre, le parquet a invoqué les outils inutiles sur un sous-marin embarqués par l&#x27;accusé peu de temps avant les faits, dont une scie à bois, des sangles de valise et un tournevis affûté de 50 cm de long.<br/><br/><h3>Rappel: Madsen &quot;désolé&quot;</h3><br/><br/>&quot;Je suis terriblement triste pour les proches de Kim de ce qui s&#x27;est passé&quot;, avait déclaré Peter Madsen à la clôture de l&#x27;audience, utilisant un adjectif qui en danois peut également signifier &quot;désolé&quot;. Le procureur Kristian Kirk avait requis la prison à vie. &quot;Quand je regarde Peter Madsen (...) je ne vois pas de raison pour (lui infliger) une peine plus faible que la perpétuité&quot;. Selon lui, la préméditation et la nature des violences qui lui sont reprochées justifiaient son enfermement à vie.<br/><br/><h3>&quot;Meurtrier sexuel cynique&quot;</h3><br/><br/>&quot;Nous parlons d&#x27;un meurtrier sexuel cynique, pervers et calculateur et pour cette raison je ne pense pas qu&#x27;il y ait une marge d&#x27;hésitation&quot;, avait souligné M. Kirk. &quot;Par défaut, tous les meurtres sont cyniques, ce n&#x27;est pas propre à cette affaire&quot;, avait rétorqué l&#x27;avocate de Peter Madsen, Betina Hald Engmark. &quot;La défense est d&#x27;avis que le bon niveau de peine se situe entre 14 et 16 ans&quot;, avait-elle martelé lors de sa plaidoirie.
	</div>
<hr/>
</div>
</body>
